chart/templates/postgres-minimal.yaml

```diff
@@ -15,4 +15,34 @@ spec:
     {{- toYaml .Values.postgresql.databases | nindent 4 }}
   postgresql:
     version: {{ .Values.postgresql.version | quote }}
+  sidecars:
+    - name: "exporter"
+      image: {{ .Values.metrics | quote }}
```

Comment:
This needs to be `.Values.metrics.image` right?

Reply:
Yes it does, fixed.